.eslintrc.json

```diff
@@ -4,11 +4,21 @@
     "node": true,
     "es6": true
   },
-  "extends": ["plugin:vue/recommended"],
-  "plugins": ["prettier"],
+  "extends": [
+    "plugin:vue/recommended",
+    "plugin:vue/essential",
+    "eslint:recommended",
+    "plugin:prettier/recommended"
+  ],
+  "plugins": [
+    "jest",
```

Comment:
Ha this highlights a stale dependency. There are no jest unit tests so we can omit this and actually remove jest as a dev dependency altogether